poncho/src/poncho/package_serverize.py

```diff
@@ -307,10 +305,10 @@ def sort_spec(spec):
     return json.dumps(sorted(conda_deps) + nested_deps, sort_keys=True).encode("utf-8")
 
 def search_env_for_spec(envpath):
+    env_spec = None
     if os.path.exists(envpath) and envpath.endswith(".tar.gz"):
-        print("Cached environment found, checking if it is compatiable with new network function")
+        print("Cached environment found, checking if it is compatiable with new library code")
```

Comment:
You may want to standardize all messages to use "library code" rather than "network function".

Reply:
I will change all taskvine messages to use Library code, but leave all the work_queue messages as network function to keep consistency.